### PR TITLE
Updates to support isort 5.0

### PR DESCRIPTION
**Pull request by cwdavies**
Removed `not_skip` and `recursive` from `tox.ini` to fix
TypeError: __init__() got an unexpected keyword argument 'not_skip'

## recursive
Prior to version 5.0.0, isort wouldn't automatically traverse directories. The --recursive option was necessary to tell it to do so. In 5.0.0 directories are automatically traversed for all Python files, and as such this option is no longer necessary and should simply be removed.

## not_skip
In an earlier version isort had a default skip of __init__.py. To get around that many projects wanted a way to not skip __init__.py or any other files that were automatically skipped in the future by isort. isort no longer has any default skips, so if the value here is __init__.py you can simply remove the setting.

https://timothycrosley.github.io/isort/docs/upgrade_guides/5.0.0/

Short description explaining the high-level reason for the pull request

## Checklist

* [x] Changes are limited to a single goal (no scope creep)
* [x] Code can be automatically merged (no conflicts)
* [x] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [x] Passes all existing automated tests

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 
 install_requires = [
-    "wagtail>=2.5,<2.6",
+    "wagtail~=2.0",
```
Comment (Scotchester) [inline setup.py:5]:
```suggestion
    "wagtail>=2.5,<2.6",
```

Let's revert this change, especially since Wagtail will soon no longer be required in this repo.

Comment (cwdavies) [reply setup.py]: Suggested change is done. Another PR will remove requirements on wagtail in this repo.

**Comment by Scotchester** [comment]
Merging this with failing back-end test because these tests are soon to be deleted from this repo, anyway.
